DOCUMENTATION.md

```diff
@@ -0,0 +1,155 @@
+All system configurations | `--filesystem=host-etc` and `--nofilesystem=host-etc` | Toggle | Permit (`--filesystem=host-etc`) or prohibit (`--nofilesystem=host-etc`) read-write access to the application to system configurations located in `/etc`. Since this directory requires root access to write, the permission will be read-only.
+All user files | `--filesystem=home` and `--nofilesystem=home` | Toggle | Permit (`filesystem=home`) or prohibit (`--nofilesystem=home`) read-write access to the application to the user directory (`$HOME`).
+Other files | `--filesystem=[PATH]`, `--filesystem=[PATH]:ro` and `--nofilesystem=[PATH]` | Input | Permit (`--filesystem=[PATH]`) or prohibit (`--nofilesystem=[PATH]`) read-write access to the application to the directory you desire. <br /> <br /> For example, you would put `~/games` if you want read-write access to `~/games`. If you want read-only (`ro`) access to `~/games`, then it will be `~/games:ro`.
+
+#### Persistent
+
+List of the homedir-relative paths created in the sandbox.
+
+Name | `flatpak-override` equivalent | Type | Description
+--- | --- | --- | ---
+Files | `--persist=[PATH]` | Input | Permit only to the application to have access to the targeted directory while restricting other applications from accessing it. <br /> <br /> Starting from `$HOME`, the targeted directory will be remapped to the application's directory (`~/.var/app/$FLATPAK_APP_ID/[PATH]`) if it has no write access to the targeted directory. <br /> <br /> For example, persisting `.mozilla` will map `~/.mozilla` to `~/.var/app/org.mozilla.Firefox/.mozilla` if the Firefox Flatpak has no write access to `~/.mozilla`.
+
+#### Environment
+
+List of variables exported to the application.
+
+Name | `flatpak-override` equivalent | Type | Description
+--- | --- | --- | ---
+Variables | `--env=[VAR]=[VALUE]` | Input | Set an environment variable in the application.
+
+#### System Bus
+
+List of well-known names on the system bus.
+
+Name | `flatpak-override` equivalent | Type | Description
+--- | --- | --- | ---
+Talks | `--system-talk-name=[NAME]` | Input | Permit the application to talk to system services. <br /> <br /> For example, inputting `org.freedesktop.Notifications` will allow the application to send notifications.
+Owns | `--system-own-name=[NAME]` | Input | Permit the application to own system services.
+
+#### Session Bus
+
+List of well-known names on the session bus.
+
+Name | `flatpak-override` equivalent | Type | Description
+--- | --- | --- | ---
+Talks | `--talk-name=[NAME]` | Input | Permit the application to talk to session services. <br /> <br /> For example, inputting `org.freedesktop.Notifications` will allow the application to send notifications.
+Owns | `--own-name=[NAME]` | Input | Permit the application to own session services.
```

Comment:
I would "... under the given name."

Reply:
https://github.com/tchx84/Flatseal/pull/136/commits/670a531e11f5a227a47d505ee99cada1efcfa314